src/panels/alarm-table-react/hooks/useOpenNMSClient.ts

```diff
@@ -0,0 +1,19 @@
+import { DataSourceRef } from "@grafana/data"
+import { getDataSourceSrv } from "@grafana/runtime"
+import { ClientDelegate } from "lib/client_delegate"
+import { useEffect, useState } from 'react'
+export const useOpenNMSClient = (datasource: DataSourceRef | null | undefined) => {
```

Comment:
Looks like this could be useful elsewhere, maybe put into a top level `hooks` folder. But, can do this when something else actually needs it.